src/theory/bv/bv_solver_simple.cpp

```diff
@@ -69,9 +70,16 @@ BVSolverSimple::BVSolverSimple(TheoryState* s,
                                TheoryInferenceManager& inferMgr,
                                ProofNodeManager* pnm)
     : BVSolver(*s, inferMgr),
-      d_bitblaster(new BBProof(s)),
-      d_epg(pnm ? new EagerProofGenerator(pnm, s->getUserContext(), "")
-                : nullptr)
+      d_tcpg(
+          pnm ? new TConvProofGenerator(pnm,
+                                        nullptr,
+                                        TConvPolicy::ONCE,
+                                        TConvCachePolicy::NEVER,
```

Comment:
It'd be good to document these

Reply:
Absolutely! Suggestions?

Reply:
`TConvPolicy::ONCE` is justified by a converted term being looked at again only once (i.e., after the children are rewritten, the post-visited term may be rewritten again). Since terms are rewritten to terms that may contain themselves, using the FIXPOINT policy could lead to infinite loops.

For the caching policy it's what Andy mentioned on slakc.